common/djangoapps/student/views.py

```diff
@@ -894,6 +894,48 @@ def dashboard(request):
     return response
 
 
+def course_refund_status(request):
+    # get course key
+    from django.http import JsonResponse
+    course_id = request.GET.get('course_id')
```

Comment:
Why make `course_id` this parameter? Why can't we have the course_id as part of the URL?